helm/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: substrate-telemetry
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```

Comment:
I'm not sure what this version number is (k8s?). I don't think much thought has ever gone into the version listed in the telemetry repos. Can this field just be removed?

Reply:
It's your application version and only matters when you want to use helm to rollback to a previous version. The CI is configured in a way that it changes this version to the commit hash, so this guarantees that the appVersion is unique for each deployment and could also be used to rollback the helm chart.